src/tests/ftest/scrubber/aggregation.yaml

```diff
@@ -0,0 +1,66 @@
+hosts:
+  test_servers: 3
+  test_clients: 1
+timeout: 600
+setup:
+  start_servers_once: false
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      targets: 8
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server0.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+    1:
+      targets: 8
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31516
+      log_file: daos_server1.log
+      log_mask: DEBUG,MEM=ERR
+      env_vars:
+        - DD_MASK=mgmt,md,rebuild
+      storage: auto
+  transport_config:
+    allow_insecure: true
+agent_config:
+  transport_config:
+    allow_insecure: true
+dmg:
+  transport_config:
+    allow_insecure: true
```

Comment:
Curious, is there a reason for `allow_insecure: true`? Can we just remove it?

Reply:
Sure. Will do...

Reply:
Done.

Reply:
Did you mean to remove this?